### PR TITLE
Zende aditya patch 2

### PR DESCRIPTION
**Pull request by ZendeAditya**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5324"></a>

### CODE REVIEW

File: data/ZendeAditya.json [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-  "bio": "A freelance web developer",
+  "bio": "**A freelance web developer** Technical Writer",
+  "tags":[
+  "open source",
+  "Web developer",
+  "Technical writer",
+  "javascript",
+  "React",
+  "Tailwind"
+  ]
```
Comment (ChinmayMhatre) [inline data/ZendeAditya.json:12]:
```suggestion
  ],
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks good! Just a missing semi colon to fix the syntax.

File: data/ZendeAditya.json (hunk 1 above)
Comment (yrncollo) [reply data/ZendeAditya.json:12]: Yeah that's the only issue and everything would be fine

**Comment by kumarsonsoff3** [comment]
Duplicate of #5328

**Pull request closed without merging**